week-1/C-hello-world/exercise.js

```diff
@@ -1 +1,4 @@
-console.log("Hello world");
+console.log("Testing Javascript " + x + " for fun.");
+
+var x=6 
```

Comment:
Understand it's just for practice, but it would be good to remember that from the start - naming variables for what they actually are is very important. What else could you call `x`?